src/components/sbb-header-action/sbb-header-action.scss

```
@@ -57,25 +62,41 @@ $breakpoints: 'zero', 'micro', 'small', 'medium', 'large', 'wide', 'ultra';
   @include sbb.button-reset;
   @include sbb.text-inherit;
 
-  text-decoration: none;
   display: flex;
+  position: relative;
   align-items: center;
   justify-content: center;
-  white-space: nowrap;
-  background-color: var(--sbb-header-action-background-color);
+  color: var(--sbb-color-black-default);
+  text-decoration: none;
   min-height: var(--sbb-header-action-min-height);
   min-width: var(--sbb-header-action-min-width);
   padding-inline: var(--sbb-header-action-padding-inline);
-  border-radius: var(--sbb-border-radius-16x);
-  transition-property: background-color;
-  transition-duration: var(--sbb-animation-duration-6x);
-  transition-timing-function: ease;
   cursor: pointer;
-  gap: var(--sbb-spacing-fixed-2x);
 
   &:focus-visible {
     @include sbb.focus-outline;
   }
+
+  &::before {
+    position: absolute;
+    content: '';
+    inset: var(--sbb-header-action-inset);
+    border-radius: var(--sbb-border-radius-16x);
```

I suggest to use the height as it is a little bit less just a fantasy value.
```suggestion
    border-radius: var(--sbb-header-action-min-height);
```